todoman/cli.py

```diff
@@ -156,7 +157,8 @@ def _todo_property_options(command):
     @functools.wraps(command)
     def command_wrap(*a, **kw):
         kw['todo_properties'] = {key: kw.pop(key) for key in
-                                 ('due', 'start', 'location', 'priority')}
```

Comment:
This bit of code actually applies to both the `new` and `edit` commands, but the `new` command already allows passing a description (I think it's the only thing that can be edited in `new`, but not in `edit`), so this approach isn't exactly right for making the description editable.